### PR TITLE
ci: add prerelease deployments to npm

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

**Review by vicb — APPROVED** [review]
🎉

File: builder/README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 
-## Build your app
+## Configure your app
 
 - update the `next.config.mjs` as follows
 
   ```typescript
   /** @type {import('next').NextConfig} */
   const nextConfig = {
-    output: "standalone",
```
Comment (IgorMinar) [inline builder/README.md:10]: is this change right?

**Review by IgorMinar — APPROVED** [review]
yay! thanks! left one q

File: builder/README.md (hunk 1 above)
Comment (vicb) [reply builder/README.md:10]: Yep, we use an env flag while building next, no need for that.
(The build output stays the same - the only exception is the addition of the `standalone` folder containing the traced files + a minial server.js)

Comment (IgorMinar) [reply builder/README.md:10]: ah. ok. thanks

Comment (IgorMinar) [reply builder/README.md:10]: I love that we don't need to change the config. great work everyone!

**Pull request merged**